test/hotspot/jtreg/runtime/cds/appcds/sharedStrings/IncompatibleOptions.java

```diff
@@ -108,9 +108,9 @@ public static void test(String[] args_ignored) throws Exception {
             testDump(1, "-XX:+UseZGC", "-XX:-UseCompressedOops", null, false);
         }
 
-        // incompatible GCs
-        testDump(2, "-XX:+UseParallelGC", "", GC_WARNING, false);
-        testDump(3, "-XX:+UseSerialGC", "", GC_WARNING, false);
+        // ParallelGC and SerialGC now supports dumping heap objects
```

Comment:
The "now" will be dated very soon. The comment is redundant really.

But do we not need other tests for Serial/Parallel and other flags combinations (or do they not apply)?